### DISCUSSION TITLE
Consider flatpak

### DISCUSSION THREAD

**Discussion by calhix** [ashutoshgngwr/noice#1117] [Ideas]
I gotta say l love the app and have been using it since the first full release. I thought it would be nice to have a Linux release (arm64 and amd64) since the main other option we have is Blanket and the sounds/UI of Notice is much better.

**Comment by ashutoshgngwr** [comment]
@calhix I have a web prototype ready to fiddle with. App: https://app.trynoice.com Source: https://github.com/trynoice/web-app-v0

**Reply by calhix** [reply]
That's actually what prompted me to post this even though this is nearly as good as it gets! I would like something akin as to what v1.3.3 is (no internet connection required) but optimized for pinephone and desktop but this is still really good!

**Reply by ashutoshgngwr** [reply]
@calhix Although I am pursuing this full-time, I have considerably limited resources. It's not enough to sustain a development effort towards multiple native apps.[^1] Native apps, even when built with multiplatform technologies, will require significant development effort, in contrast to maintaining the existing Android app and building a web app. The only foreseeable downside is the missing offline capability.

Although it isn't entirely impossible to make a web app work offline, e.g. my [personal blog](https://ashutoshgngwr.github.io) works offline, it might be difficult. It is something I don't intend to explore right away. Building a functional web app takes precedence for now.

[^1]: I've also received requests (via emails and #598) for iOS, Mac and Windows apps. It is what prompted me to build a web app.
